docs/reference/objects/product/extra.md

```diff
@@ -53,6 +53,10 @@ Returns the initial stock for the extra, if the extra has unlimited inventory th
 
 Returns a list of [modifier groups](({% link docs/reference/objects/product/modifier_group.md %})) associated with this extra.
 
```

Comment:
Since we are changing this method, have noticed that this link is broken. Have too many brackets around the link, can we add a commit to fix it as well?